pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminImpl.java

```diff
@@ -107,24 +106,6 @@
     private final int requestTimeout;
     private final TimeUnit requestTimeoutUnit;
 
-    static {
-        /**
-         * The presence of slf4j-jdk14.jar, that is the jul binding for SLF4J, will force SLF4J calls to be delegated to
-         * jul. On the other hand, the presence of jul-to-slf4j.jar, plus the installation of SLF4JBridgeHandler, by
-         * invoking "SLF4JBridgeHandler.install()" will route jul records to SLF4J. Thus, if both jar are present
-         * simultaneously (and SLF4JBridgeHandler is installed), slf4j calls will be delegated to jul and jul records
-         * will be routed to SLF4J, resulting in an endless loop. We avoid this loop by detecting if slf4j-jdk14 is used
-         * in the client class path. If slf4j-jdk14 is found, we don't use the slf4j bridge.
-         */
-        try {
-            Class.forName("org.slf4j.impl.JDK14LoggerFactory");
-        } catch (Exception ex) {
-            // Setup the bridge for java.util.logging to SLF4J
-            SLF4JBridgeHandler.removeHandlersForRootLogger();
-            SLF4JBridgeHandler.install();
```

Comment:
This was to route java.util.logging into SLF4J and it's due to the unfortunate fact that Jersey client is using JUL. If we remove it from here, I thing that Jersey logs will not be properly routed.

Reply:
I'll return this back. I thought it was there to workaround the issue caused by slf4j-jdk14.jar . slf4j-jdk14 routes SLF4J to JUL. There's already JUL-to-SLF4J in the Pulsar libraries.

Reply:
Reverted back to original

Reply:
Yes, this is to initialize the "JUL-to-SLF4J" bridge, but only if you're not using JUL as your final implementation.